xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -707,7 +707,6 @@ bool CVideoPlayer::OpenFile(const CFileItem& file, const CPlayerOptions &options
   }
 
   m_item = file;
-  m_callback.OnPlayBackStarted(m_item);
```

Comment:
this has to be kept.

Reply:
May I ask why it cannot wait for Prepare()?

Reply:
It would leave the app in an undefined state for up to a couple of seconds (this long may it take Pepare to execute). As soon a user clicks on a video item, playback starts, currentFileItem of CApp is set, and the app is aware that player is active.